.flake8

```diff
@@ -32,7 +32,7 @@ ignore =
     # no warnings about pickle,
     S301,S403,
     # no warnings about XML parsing
-    S318,S320,S405,S408,S410,
+    S314,S318,S320,S405,S408,S410,
```

Comment:
What warning is `S314`?

Reply:
parsing untrusted XML. In case we use `etree` this would only mean DoS attacks like billion laughs/ quadratic blowup(cf. https://laconicwolf.com/2018/08/13/writing-and-fixing-vulnerable-xml-parsing-code-in-python/ ), no ACE.

I don't see the parser call though, is probably in code not touched by this PR? In general I would not white list this warning per se in `.flake8` and instead on a per call basis at the location where the XML is parsed. Otherwise we might miss cases in which the warning would actually be justified.

Reply:
More specifically, S314 warns about parsing untrusted XML via `xml.etree.ElementTree`.
This warning appears for the `fromstring` calls in `test_mergeBenchmarkSets.py`.
I mainly added this to the `.flake8` file seeing that S318 and S320 are already there as well which are basically the same for `xml.dom.minidom` and `lxml.etree` respectively, but putting `#noqa` to each call site is no problem -> done.